### DISCUSSION TITLE
Changing colors of vertex markers in edit mode

### DISCUSSION THREAD

**Discussion by victoriaunizhona** [geoman-io/leaflet-geoman#850] [Questions & Help]
Hi guys!

I need to set color of these markers(orange ones) in edit mode for polygon dynamically. Did not found the right method in documentation.

I can set the fill color for polygon, lines, but no luck with markers. Is it possible?

Would really appreciate any help.

![image](https://user-images.githubusercontent.com/32535863/105990728-93c88c00-60ab-11eb-9c25-9cbf1c86e3a7.png)

**Comment by Falke-Design** [comment]
This markers are only styled by css, but I think you have found this already because of the change to orange.

https://github.com/geoman-io/leaflet-geoman/blob/7cdb0edadc800edf3498fe6f78d140bccaa0aaa9/src/css/layers.css#L1-L12


You can add a css class to the map and if the class is applied to the map change the css of the vertex markers:

https://jsfiddle.net/falkedesign/ojg3v0dt/
```
var changed = false;
function changeColor(){
  if(!changed){
    L.DomUtil.addClass(map.getContainer(),'custom-vertex');
  }else{
    L.DomUtil.removeClass(map.getContainer(),'custom-vertex');
  }
  changed = !changed;
}
```

CSS:
```
 #map.custom-vertex .marker-icon { 
   background-color: #f00;  
 } 
```
